app/models/api/transliterate.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+// Check that the class exists before trying to use it
+if (! class_exists('Transliterator')) {
+    $request->error = 'Class Transliterator not present';
+    $json = $request->invalidAPICall();
+
+    return;
+}
+
+switch ($request->parameters[2]) {
+    case 'serbian':
+        $transliterated_locale = 'Serbian-Latin/BGN';
+        break;
+
+    default:
+        break;
+}
+
+if (! $transliterated_locale) {
```

Comment:
this is throwing a warning when calling the API with a wrong locale, we should use instead:
if (! isset($transliterated_locale))